### ISSUE TITLE
Client won't start minimized

### ISSUE THREAD

**Issue by arebalof** [WeakAuras/WeakAuras-Companion#1015]
**Operating System:**
- [ x] Windows
- [ ] Mac
- [ ] Linux

**Companion Version:**
```
3.2.2
```

**Companion App Console Log:**
```
electron/js2c/renderer_init.js:13 (electron) The remote module is deprecated. Use https://github.com/electron/remote instead.
log @ electron/js2c/renderer_init.js:13
app.0cb7ee51.js:1 validateWowpath
app.0cb7ee51.js:1 buildVersionList
2app.0cb7ee51.js:1 buildAccountList
app.0cb7ee51.js:1 updaterHandler: update-not-available
app.0cb7ee51.js:1 updaterHandler: checkForUpdates
app.0cb7ee51.js:1 promisesWagoCallsComplete
app.0cb7ee51.js:1 promisesWagoDataCallsComplete
app.0cb7ee51.js:1 no data received for InterruptTracker
app.0cb7ee51.js:1 no data received for PartyCDs
app.0cb7ee51.js:1 writeAddonData
```

**WeakAuras Version:**
```
/run print(WeakAuras.versionString) in-game.
```

**WeakAuras Errors:**
```
1. Install https://mods.curse.com/addons/wow/BugSack
2. Install https://mods.curse.com/addons/wow/bug-grabber
3. Try to do whatever it is you're having a problem with again
4. Check your minimap for an icon that looks like a bag.  Is it green, or is it red?
   • GREEN
      • WeakAuras is working correctly.
      • Either you found a behavioral bug, or you are just doing it wrong :)
      • Explain the steps you took, and we will try to figure out what happened.
   • RED 
     • There are error messages. Click on the red bag to see them.
     • Copy & Paste here all of the error messages (there may be more than one, click the 'previous' button on the dialog to see previous ones
```

**Describe the bug**
Client doesn't start minimized after updating to 3.2.2 despite the option being checked. Tried to check/uncheck. Also tried to uncheck, restart, check again, restart.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Include a .zip file containing:**
`%programfiles%\World of Warcraft\_retail_\WTF\Account\<AccountName>\SavedVariables\WeakAuras.lua`
AND
`%appdata%\weakauras-companion\config.json`

**Comment by Batmanerman** [comment]
Having this exact same issue as well and came here to report it.

**Comment by Stanzilla** [comment]
It is fixed already but we haven't released a new version yet, so coming soon :)

**Comment by mrbuds** [comment]
This should be fixed in version 3.3.1
